src/templates/specific-project.js

```diff
@@ -22,11 +24,27 @@ const ProjectTemplate = ({ data }) => (
                 <Link style={{color: '#1C2833', paddingBottom: '20px'}} to={`/${data.strapiProject.profiles.profile_name}`}>Sponsored by {data.strapiProject.profiles.profile_name}</Link>
                 <br/>
                 <br/>
-                <Link className="button has-text-centered" href='/' type="default" block>
-                  Contact Sponsor
-                </Link>
+                <br/>
+              <div className="columns is-centered is-flex" >
```

Comment:
I don't like the look of the icons being centered. Can we get rid of the `is-centered` and the entire empty column div?